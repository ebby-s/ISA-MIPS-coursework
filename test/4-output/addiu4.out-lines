TB : INFO : register_v0=fffffc01
TB : finished; active=0
