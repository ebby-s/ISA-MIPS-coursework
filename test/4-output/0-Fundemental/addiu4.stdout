Loading test/2-binary/0-Fundemental/addiu4.hex.txt into RAM
TB : INFO : RAM_ACCESS: Read from 0xbfc00000, data: 0x2442fe37
TB : INFO : RAM_ACCESS: Read from 0xbfc00004, data: 0x2442fdca
TB : INFO : RAM_ACCESS: Read from 0xbfc00008, data: 0x0000f808
TB : INFO : RAM_ACCESS: Read from 0xbfc0000c, data: 0x00000000
TB : INFO : RAM_ACCESS: Read from 0x00000000, data: 0x00000000
TB : INFO : register_v0=fffffc01
TB : Finished : active=0
